test_cases/structured_geocoding.json

```diff
@@ -1065,6 +1083,150 @@
           }
         ]
       }
+    },
+
+    {
+      "id": 1100,
+      "status": "pass",
+      "user": "trescube",
+      "type": "dev",
+      "notes": "query+address should ignore address",
```

Comment:
Otherwise we'll search for that query at exactly that address.  That is, it would only find "police" at "1090 N Charlotte St" whereas we want to search within the greater geographic context.